examples/tts/conf/vits.yaml

```diff
@@ -0,0 +1,256 @@
+        apostrophe: True
+        sep: '|'
+        add_blank_at: null
+        pad_with_space: True
+        g2p:
+          _target_: "nemo.collections.tts.torch.g2ps.EnglishG2p"
+          phoneme_dict: ${phoneme_dict_path}
+          heteronyms: ${heteronyms_path}
+
+    dataloader_params:
+      drop_last: false
+      shuffle: true
+      batch_size: 2
+      num_workers: 1
+      pin_memory: false
+
+
+  preprocessor:
+    _target_: nemo.collections.asr.parts.preprocessing.features.FilterbankFeatures
+    dither: 0.0
+    nfilt: ${model.n_mel_channels}
+    frame_splicing: 1
+    highfreq: 8000
+    log: true
+    log_zero_guard_type: clamp
+    log_zero_guard_value: 1e-05
+    lowfreq: 0
+    mag_power: 1.0
+    n_fft: 1024
+    n_window_size: ${n_window_size}
+    n_window_stride: ${model.hop_size}
+    normalize: null
+    pad_to: 1
+    pad_value: 0
+    preemph: null
+    sample_rate: ${model.sample_rate}
+    window: hann
+    exact_pad: true
+    use_grads: false
+
+  input_fft:
+    _target_: nemo.collections.tts.modules.transformer.FFTransformerEncoder
+    n_layer: 6
+    n_head: 1
+    d_model: ${model.symbols_embedding_dim}
+    d_head: 64
+    d_inner: 1536
+    kernel_size: 3
+    dropout: 0.1
+    dropatt: 0.1
+    dropemb: 0.0
+    n_embed: 148  # NOTE Should match # of tokens in `symbol_set`
+    d_embed: ${model.symbols_embedding_dim}
+    padding_idx: 0
+
+  output_fft:
+    _target_: nemo.collections.tts.modules.transformer.FFTransformerDecoder
+    n_layer: 6
+    n_head: 1
+    d_model: ${model.symbols_embedding_dim}
+    d_head: 64
+    d_inner: 1536
+    kernel_size: 3
+    dropout: 0.1
+    dropatt: 0.1
+    dropemb: 0.0
+
+  duration_predictor:
+    _target_: nemo.collections.tts.modules.vits_modules.StochasticDurationPredictor
+    in_channels: ${model.symbols_embedding_dim} # input_size: ${model.symbols_embedding_dim}
+    kernel_size: 3
+    filter_channels: ${filter_length} # filter_size: 256
+    p_dropout: 0.1 # dropout: 0.1
+    # n_layers: 6
+
+  pitch_predictor:
+    _target_: nemo.collections.tts.modules.fastpitch.TemporalPredictor
+    input_size: ${model.symbols_embedding_dim}
+    kernel_size: 3
+    filter_size: ${filter_length}
+    dropout: 0.1
+    n_layers: ${model.n_layers}
+
+  generator:
+    _target_: nemo.collections.tts.modules.vits_modules.Generator
+    resblock: "1"
+    resblock_kernel_sizes: [3,7,11]
+    resblock_dilation_sizes: [[1,3,5], [1,3,5], [1,3,5]]
+    upsample_rates: [8,8,2,2]
+    upsample_initial_channel: 512
+    upsample_kernel_sizes: [16,16,4,4]
+    initial_channel: 384 # initial_input_size: 384
+
+trainer:
+  gpus: -1 # number of gpus
+  max_epochs: 20000
+  num_nodes: 1
+  accelerator: ddp
+  accumulate_grad_batches: 1
+  checkpoint_callback: False  # Provided by exp_manager
+  logger: False  # Provided by exp_manager
+  # gradient_clip_val: 1000.0
+  flush_logs_every_n_steps: 1000
+  log_every_n_steps: 100
+  check_val_every_n_epoch: 5
+  precision: 16
```

Comment:
FYI, you should use `precision: 16`, only if you are using a Volta or Ampere GPU. Otherwise, it's better to remove this line

Reply:
Removed in df7e99660e5c7f957fdbec153dbeb1b239579ce6